Makefile

```diff
@@ -59,6 +59,11 @@ bin/examples/pulumi-resource-%: $$(shell $${HELPMAKEGO} examples/$$*)
 examples/%/schema.json: bin/examples/pulumi-resource-%
 	pulumi package get-schema ./$< > $@
 
+# Generate a Go SDK from an example provider schema
+.PHONY: examples/%/gen-sdk
```

Comment:
Would it make sense to run this before running the examples test target as well? It seems like a pre-req to have the SDKs up to date to ensure that the example test will pass.

Reply:
I had considered that, and decided against it, but I suppose it would be harmless.